src/main/java/com/smartbear/readyapi/plugin/git/ui/ImportProjectFromGitGui.java

```diff
@@ -25,8 +26,13 @@ public Component getComponent() {
     @Override
     public VcsRepositoryInfo downloadProjectFiles(File emptyDirectory) {
         try {
-            gitCommandHelper.cloneRepository(getSelected().getRepositoryPath(), getSelected().getCredentialsProvider(), emptyDirectory);
-            return new VcsRepositoryInfo("Git", getSelected().getRepositoryPath());
+            String repositoryPath = getSelected().getRepositoryPath();
+            if (isLocal() || StringUtils.isNullOrEmpty(repositoryPath)) {
+                return null;
```

Comment:
I think, we can read appropriate information from .git directory and return a valid VcsRepositoryInfo object instead of null.

Reply:
I've done it. But this code is not called from ReadyAPI if the repository is local.